sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

```diff
@@ -172,19 +183,24 @@ def begin_recognize_content_from_url(self, url, **kwargs):
 
     @distributed_trace
     def begin_recognize_custom_forms(self, model_id, stream, **kwargs):
-        # type: (str, IO[bytes], Any) -> LROPoller
-        """Analyze Form.
+        # type: (str, Union[bytes, IO[bytes]], Any) -> LROPoller
+        """Analyze a custom form with a model trained with or without labels.
+        The input document must be of one of the supported content types - 'application/pdf',
+        'image/jpeg', 'image/png' or 'image/tiff'.
 
         :param str model_id: Model identifier.
```

Comment:
I'd call this "Custom model identifier" to keep it consistent with how we're addressing it elsewhere